src/app/project/register/page.tsx

```
@@ -12,6 +12,7 @@ export default function ProjectRegister() {
   const [currentStep, setCurrentStep] = useState(1)
   const [formData, setFormData] = useState({
     targetAmount: "",
+    supportOptions: [] as any[],
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)